src/client/elm/Config.elm

```diff
@@ -77,27 +77,20 @@ developmentUrl mode localUrl =
             localUrl
 
         Staging ->
-            "https://dashboard.eu.staging.concordium.com"
+            "https://dashboard.eu.staging.concordium.software"
 
         Testnet ->
-            "https://dashboard.testnet.concordium.com"
+            "https://dashboard.testnet.concordium.software"
```

Comment:
This is not correct at this point in time. It might be in the future, but currently it is still `.com`.